src/load-npm-tasks-properly.js

```diff
@@ -0,0 +1,21 @@
+"use strict";
+var fluid = require("infusion");
+var gpii = fluid.registerNamespace("gpii");
+
+var resolve = require("fluid-resolve");
+
+fluid.registerNamespace("gpii.grunt.lintAll");
+gpii.grunt.lintAll.fixGruntTaskLoading = function (grunt) {
+    grunt.loadNpmTasksProperly = function (name) {
```

Comment:
Whilst the comment for this method needn't be as ad hominem as the original one, we should have some kind of fairly detailed explanation of what the problem is here and why this approach is a solution to it

Reply:
OK, it's definitely gonna be detailed, as there are two problems, and our fix only addresses one.  I'll take a stab at it and you can comment on that shortly.